.gitignore

```diff
@@ -23,5 +23,8 @@ build
 test/results
 flow
 
+.idea/
+cmake-build-debug/
+
```

Comment:
Are these needed?

Reply:
I've been trying out CLion and the default configuration creates these folders that should be ignored.